### ISSUE TITLE
Completion popup not always visible.

### ISSUE THREAD

**Issue by kwinczek** [Shougo/deoplete.nvim#482]
# Problems summary

Deoplete shows popup menu only once every other character is typed in.

## Expected

Popup menu to stay visible all the time when completions are available,

## Environment Information

 * deoplete version(SHA1): eeda83b2234844bea5027d22a3e3d1804f50f758

 * OS: Mac OS 10.12.5

 * neovim version: 0.2.0

 * `:CheckHealth` result(neovim ver.0.1.5-452+):

```
health#deoplete#check
========================================================================
## deoplete.nvim
  - SUCCESS: has("nvim") was successful
  - SUCCESS: has("python3") was successful
  - INFO: If you're still having problems, try the following commands:
    $ export NVIM_PYTHON_LOG_FILE=/tmp/log
    $ export NVIM_PYTHON_LOG_LEVEL=DEBUG
    $ nvim
    $ cat /tmp/log_{PID}
    and then create an issue on github

health#nvim#check
========================================================================
## Configuration
  - SUCCESS: no issues found

## Performance
  - SUCCESS: Build type: RelWithDebInfo

## Remote Plugins
  - SUCCESS: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=\177
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~

health#provider#check
========================================================================
## Clipboard (optional)
  - SUCCESS: Clipboard tool found: pbcopy

## Python 2 provider (optional)
  - INFO: `g:python_host_prog` is not set.  Searching for python2 in the environment.
  - INFO: Executable: /usr/local/bin/python2
  - INFO: Python2 version: 2.7.13
  - INFO: python2-neovim version: 0.1.13
  - SUCCESS: Latest python2-neovim is installed: 0.1.13

## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Executable: /usr/local/bin/python3
  - INFO: Python3 version: 3.6.1
  - INFO: python3-neovim version: 0.1.13
  - SUCCESS: Latest python3-neovim is installed: 0.1.13

## Ruby provider (optional)
  - INFO: Ruby: ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-darwin16.4.0]
  - WARNING: Missing "neovim" gem.
    - SUGGESTIONS:
      - Run in shell: gem install neovim
      - Is the gem bin directory in $PATH? Check `gem environment`.
      - If you are using rvm/rbenv/chruby, try "rehashing".
```

## Provide a minimal init.vim with less than 50 lines (Required!)

```vim
call plug#begin('~/.config/nvim/minimal-plugged')

Plug 'Shougo/deoplete.nvim'

call plug#end()

inoremap <expr> <CR> pumvisible() ? "\<C-y>" : "\<CR>"

let g:deoplete#enable_at_startup = 1
let g:deoplete#enable_refresh_always = 1
let g:deoplete#enable_ignore_case = 1
let g:deoplete#enable_smart_case = 1
let g:deoplete#enable_camel_case = 1
let g:deoplete#file#enable_buffer_path = 1
call deoplete#custom#set('_', 'matchers', ['matcher_full_fuzzy'])
set omnifunc=syntaxcomplete#Complete
set completeopt=longest,menuone,preview,noinsert
```

## The reproduce ways from neovim starting (Required!)

1. Start nvim with aforementioned minimal configuration file.
2. Type in any word.
3. Type it again to see pop up menu disappear once every other character is typed in.

## Screen shot (if possible)

![completion](https://cloud.githubusercontent.com/assets/796021/26250614/49a97b68-3cb3-11e7-8ad1-4cfffb9e3f0f.gif)

**Comment by z33ky** [comment]
This is still an issue for me. I am on the latest commit (04de49e2a7480ac9142fc38faba0ed62a9547c32).

**Comment by Shougo** [comment]
@z33ky We have not ESP skills.  Please upload the minimal vimrc.

**Comment by z33ky** [comment]
Seemed to be a transient error as it disappeared after attempting to create a minimal vimrc, where I could not replicate the issue.
Sorry for the noise.

**Comment by Shougo** [comment]
OK

**Comment by sebastien** [comment]
I confirm this issue with Ubuntu 17.10 stock nvim (NVIM v0.2.0), tested using the minimal configuration.

**Comment by Shougo** [comment]
@sebastien Please create the new issue.
Otherwise, I will ignore it.
And please test parallel branch instead.
